### PR TITLE
Dev/v1.2

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Review by oliver-walker-hms — DISMISSED** [review]

**Review by TomKimsey — CHANGES_REQUESTED** [review]
It would be beneficial to have more class documentation for DataPointIntegerEnum. Maybe even giving an example or reason for why to use it. Also the class name seems a bit confusing. Isnt the meaningful value datatype really a string and not an int?

ex
0 = "Off"
1= "On, Not Running"
2 = "On, Running"
3 = "On, Not Running, Faulted"

To me string better defines those datatypes than integer. Of maybe the class should be something more like DataPointIntegerEncodedString

Edit: Encoded is probably not the right word to use 
Maybe DataPointIntegerMappedString

**Comment by alexjhawk** [comment]
> It would be beneficial to have more class documentation for DataPointIntegerEnum. Maybe even giving an example or reason for why to use it. Also the class name seems a bit confusing. Isnt the meaningful value datatype really a string and not an int?
> 
> ex
> 0 = "Off"
> 1= "On, Not Running"
> 2 = "On, Running"
> 3 = "On, Not Running, Faulted"
> 
> To me string better defines those datatypes than integer. Of maybe the class should be something more like DataPointIntegerEncodedString
> 
> Edit: Encoded is probably not the right word to use
> Maybe DataPointIntegerMappedString

- Renamed new data point class to your suggested name "DataPointIntegerMappedString" because I feel that fits well. 
- Added example in class documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Flexy Data Point Library Changelog
+
+## Version 1.2
+### Features
+- Added integer enumeration data point type
```
Comment (TomKimsey) [inline CHANGELOG.md:5]: needs to get updated for new class name

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alexjhawk) [reply CHANGELOG.md]: updated

**Review by TomKimsey — APPROVED** [review]

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**